test/unittests/test_utils.py

```diff
@@ -24,7 +24,7 @@ def setup_function():
     utils._ip_for_cloud = None
     # Mock memoize method and reload the module under test later with imp
     mock.patch('crmsh.utils.memoize', lambda x: x).start()
-    imp.reload(utils)
+    importlib.reload(utils)
```

Comment:
It is better to use `functools.wraps` and patch `crmsh.utils.memoize.__wrapped__`, to avoid relying on reloading the module.

Reply:
Changed